exchanges/binanceus/binanceus.go

```diff
@@ -0,0 +1,2001 @@
+	return resp,
+		bi.SendHTTPRequest(ctx, exchange.RestSpotSupplementary, path, rateLimit, &resp)
+}
+
+// GetPriceChangeStats returns price change statistics for the last 24 hours
+// symbol: string of currency pair
+func (bi *Binanceus) GetPriceChangeStats(ctx context.Context, symbol currency.Pair) (PriceChangeStats, error) {
+	resp := PriceChangeStats{}
+	params := url.Values{}
+	rateLimit := spotPriceChangeAllRate
+	if !symbol.IsEmpty() {
+		rateLimit = spotDefaultRate
+		symbolValue, err := bi.FormatSymbol(symbol, asset.Spot)
+		if err != nil {
+			return resp, err
+		}
+		params.Set("symbol", symbolValue)
+	}
+	path := common.EncodeURLValues(priceChange, params)
+
+	return resp, bi.SendHTTPRequest(ctx,
+		exchange.RestSpotSupplementary, path, rateLimit, &resp)
+}
+
+// GetTickers returns the ticker data for the last 24 hrs
+func (bi *Binanceus) GetTickers(ctx context.Context) ([]PriceChangeStats, error) {
+	var resp []PriceChangeStats
+	return resp, bi.SendHTTPRequest(ctx,
+		exchange.RestSpotSupplementary, priceChange, spotPriceChangeAllRate, &resp)
+}
+
+// GetAccount returns binance user accounts
+func (bi *Binanceus) GetAccount(ctx context.Context) (*Account, error) {
+	type response struct {
+		Response
+		Account
+	}
+	var resp response
+	params := url.Values{}
+	if err := bi.SendAuthHTTPRequest(ctx,
+		exchange.RestSpotSupplementary,
+		http.MethodGet, accountInfo,
+		params, spotAccountInformationRate,
+		&resp); err != nil {
+		return &resp.Account, err
+	}
+
+	if resp.Code != 0 {
+		return &resp.Account, errors.New(resp.Msg)
+	}
+
+	return &resp.Account, nil
+}
+
+// GetUserAccountStatus  to fetch account status detail.
+func (bi *Binanceus) GetUserAccountStatus(ctx context.Context, recvWindow uint) (*AccountStatusResponse, error) {
+	var resp AccountStatusResponse
+	params := url.Values{}
+	timestamp := time.Now().UnixMilli()
+	params.Set("timestamp", strconv.Itoa(int(timestamp)))
+	if recvWindow > 0 && recvWindow < 60000 {
+		if recvWindow < 2000 {
+			recvWindow += 1500
+		}
+		params.Set("recvWindow", strconv.Itoa(int(recvWindow)))
+	}
+
+	return &resp,
+		bi.SendAuthHTTPRequest(ctx,
+			exchange.RestSpotSupplementary,
+			http.MethodGet,
+			accountStatus,
+			params,
+			spotAccountInformationRate,
+			&resp)
+}
+
+// GetUserAPITradingStatus to fetch account API trading status details.
+func (bi *Binanceus) GetUserAPITradingStatus(ctx context.Context, recvWindow uint) (*TradeStatus, error) {
+	type response struct {
+		Success bool        `json:"success"`
+		TC      TradeStatus `json:"status"`
+	}
+	var resp response
+	params := url.Values{}
+	timestamp := time.Now().UnixMilli()
+	params.Set("timestamp", strconv.Itoa(int(timestamp)))
+	if recvWindow > 0 && recvWindow < 2000 {
+		recvWindow += 1500
+	}
+	params.Set("recvWindow", strconv.Itoa(int(recvWindow)))
+	return &resp.TC,
+		bi.SendAuthHTTPRequest(ctx,
+			exchange.RestSpotSupplementary,
+			http.MethodGet,
+			tradingStatus,
+			params,
+			spotAccountInformationRate,
```

Comment:
This has a ratelimit weight of 1: https://docs.binance.us/#get-user-api-trading-status